docker-storage-setup.sh

```diff
@@ -823,6 +830,13 @@ setup_storage() {
   fi
 }
 
+reset_storage() {
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+	reset_lvm_thin_pool
+    fi
+    rm -f $DOCKER_STORAGE
```

Comment:
Having said that, I feel we probably should get rid of that file from docker package. It seems to be just a place holder for documentation. All the documentation should go in web pages and in /usr/share/doc/ dir. We are not expecting users to modify /etc/sysconfig/docker-storage and its generated by tool.

So I would think, that don't restore it and in a separate PR to docker, simply remove it and put relevant documentation as appropriate.